tfac/figures/figure1.py

```diff
@@ -17,7 +17,7 @@ def makeFigure():
     ax[0].axis("off")
     ax[1].axis("off")
 
-    tensor, _, _ = form_tensor()
+    tensor = form_tensor()
```

Comment:
I don't see where form_tensor has changed?

Reply:
The function was returning the times and the names of conditions, but we were not using them anywhere, so I removed those two last variables from return. Are they necessary?

Reply:
You will find them helpful when plotting the factors (in fact I believe Fig. 2 uses these).

Reply:
Will bring them back. Thanks.